.circleci/config.yml

```diff
@@ -66,7 +66,7 @@ jobs:
                 command: |
                     python -m venv venv
                     . venv/bin/activate
-                    pip install dash_bio_utils
+                    pip install .
```

Comment:
```suggestion
```
This line is unnecessary; the `dash-bio` package is installed later on in the CI job.

Reply:
We need to install dash_bio for use dash_bio.utils in node tests